### PR TITLE
ETS 1483 make whole teaser clickable

### PR DESCRIPTION
**Pull request by BirgitPohl**
### Making the whole teaser clickable.
Since there are many PR's open, I can not decide on a version yet.
We should communicate the versioning, merging and deployment.

The PR includes a refactoring regarding the buttons, which I needed for changing the type of button inside the teaser.
The collection of buttons are now shown under **pages**.

### CODE REVIEW

**Review by brunogeronimo — APPROVED** [review]

File: source/_patterns/00-atoms/10-misc/04-image-responsive.mustache [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <picture>
-    <source srcset="{{ imgSrcResp.lg }}" media="(min-width: 60em)">
-    <source srcset="{{ imgSrcResp.md }}" media="(min-width: 48em)">
-    <source srcset="{{ imgSrcResp.sm }}" media="(min-width: 30em)">
-    <source srcset="{{ imgSrcResp.xs }}">
+    <source srcset="{{ imgSrcResp.lg }}" media="(min-width: 60em)" />
```
Comment (cllagosr) [inline source/_patterns/00-atoms/10-misc/04-image-responsive.mustache:2]: `source` tags  must not have an end tag

**Review by vmorgado — APPROVED** [review]

**Review by susannegernandt — APPROVED** [review]
Version and Tag still need to be added

**Review by dknx01 — APPROVED** [review]
looks good so far.
It is a little bit hard to find, what really belongs to the tease and what to the changes of the styleguide-website. Maybe it is a little bit better to separate this in the future into two PR's? Especially if the new menu entries are not mentioned in the ticket.

**Review by cllagosr — COMMENTED** [review]
The buttons directory is huge, you should consider splitting it in smaller sets.

**Pull request closed without merging**